features/doctrine/order_filter.feature

```diff
@@ -302,7 +302,7 @@ Feature: Order filter on collections
 
   Scenario: Get collection ordered in ascending order on an association and on which order filter has been enabled in whitelist mode
     Given there is "30" dummy objects with relatedDummy
-    When I send a "GET" request to "/dummies?order[relatedDummy]=asc"
+    When I send a "GET" request to "/dummies?order[relatedDummy.name]=asc"
```

Comment:
why the change? We should do a new test if we want to test the name property no?